theme/index.hbs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE HTML>
-<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
+<html lang="{{ language }}"  class="sidebar-visible no-js {{ default_theme }}">
```

Comment:
Thanks for the revert — let's also take out the extra space so that the file is completely untouched :smile: 

```suggestion
<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
```